packages/dropzone/src/core.ts

```diff
@@ -0,0 +1,206 @@
+/**
+ * This is a forked version of the react-dropzone package, that's been minified
+ * to suit UploadThing's needs and be easily portable to other frameworks than React.
+ * See original source here: https://github.com/react-dropzone/react-dropzone
+ * The original package is licensed under the MIT license.
+ */
+
+import attrAccepts from "attr-accept";
+
+import type { AcceptProp, DropzoneState, FileWithPath } from "./types";
+
+// eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
+const accepts: typeof attrAccepts =
+  // @ts-expect-error - ESM interop
+  typeof attrAccepts === "function" ? attrAccepts : attrAccepts.default;
+
+// Firefox versions prior to 53 return a bogus MIME type for every file drag, so dragovers with
+// that MIME type will always be accepted
+export function isFileAccepted(file: File, accept: string | string[]) {
+  return file.type === "application/x-moz-file" || accepts(file, accept);
+}
+
+export function isEnterOrSpace(event: { key?: string; keyCode?: number }) {
+  return (
+    ("key" in event && (event.key === " " || event.key === "Enter")) ||
+    ("keyCode" in event && (event.keyCode === 32 || event.keyCode === 13))
+  );
+}
+
+const isDefined = <T>(v: T | null | undefined): v is T => v != null;
+export function isValidSize(file: File, minSize: number, maxSize: number) {
```

Comment:
Fixed it @t3dotgg - not sure how much of this is even needed, files should always have a size on them and minSize and maxSize are typed as numbers, so there shouldn't be a case where those are nullish either...

can probably be a one-liner

```ts
export const isValidSize = (file: File, minSz: number, maxSz: number) => file.size >= minSz && file.size <= maxSz;
```